### DISCUSSION TITLE
migrate from LAUNCHXL-CC1352P-2 to SLZB-06

### DISCUSSION THREAD

**Discussion by tabnul** [Koenkk/zigbee2mqtt#14743] [General]
I am considering to migrate from LAUNCHXL-CC1352P-2 to SLZB-06 . The latter is network attached so i wonder whether i can keep my network and devices intact without rejoining etc?

**Comment by Koenkk** [comment]
No repairing is needed for this case (https://www.zigbee2mqtt.io/guide/faq/#what-does-and-does-not-require-repairing-of-all-devices)

**Reply by tabnul** [reply]
Thank you, a case of RTFM 😬
